docs/dev/annual_updates.rst

```diff
@@ -156,57 +142,140 @@ B. FERC Form 1
+    ``reported_value``
+
+    The ``dbf_only`` column is marked ``TRUE`` if the ``row_literal`` only shows up in
+    the DBF files. An common example is when several fields are aggregated in the DBF
+    data but not in XBRL. The ``notes`` column is a place to indicate complexity or
+    reasoning and is intended for humans (vs. computers) to read.
+
+
+**4.B.2)** Use the FERC 1 debugging notebook ``devtools/ferc1-etl-debug.ipynb`` to run
+the transforms for each table. Heed any errors or warnings that pop up in the
+logs. One of the most likely bugs will be uncategorized strings. Any column listed in
+:py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS` under ``categorize_strings``
+gets cleaned according to varables defined in :mod:`pudl.transform.params.ferc1`.
+Uncategorized string values (think new, strange fuel type spellings) will throw an error
+and should be added to the appropriate dictionaries in
+:mod:`pudl.transform.params.ferc1`.
+
+**4.B.3)** If there's a new column, add it to the transform process. At the very least,
+you'll need to include it in the ``rename_columns`` dictionary in
+:py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS` for the appropriate table.
+
+* Consider whether the column could benefit from any of the standard transforms in
+  :mod:`pudl.transform.classes`. If so, add those transforms/the new column to
+  :py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS`. If the standard transform
+  you're applying to the new column was not previously listed in
+  :py:const:`pudl.transform.params.ferc1.TRANSFORM_PARAMS` under the table you're
+  working on, you'll need to add it to ``transform_main()`` in
+  :mod:`pudl.transform.ferc1` under the cooresponding table transformer class or it
+  won't be executed.
+
+* Consider whether the column could benefit from custom transformations. If it's
+  something that could be applicable to other columns, consider building it in
+  :mod:`pudl.tranform.classes`. If not, build it in the relevant table transformer
+  class in :mod:`pudl.transform.ferc1`. Make sure to write a unit test for any new
+  functions.
+
+**4.B.4)** If there's a new table, add it to the transform process. You'll need to build
+or augment a table transformer in :mod:`pudl.transform.ferc1` and follow all
+instructions applicable to new columns.
+
+**4.B.5)** To see if the transformations work, you can run the transform module as a
+script in the terminal. From within the pudl repo directory, run:
+
+.. code-block:: bash
+
+    python src/pudl/transform/ferc1.py
+
+
+5. Update the PUDL DB Schema
+----------------------------
+If new columns or tables have been added, you must also update the PUDL DB schema,
+define column types, give them meaningful descriptions, apply appropriate ENUM
+constraints, etc. This happens in the :mod:`pudl.metadata` subpackage. Otherwise when
+the system tries to write dataframes into SQLite, it will fail or simply exclude any new
+columns.
```

Comment:
I would add a note here that many of our transform processes rely on the schema! both the encoding step (mostly applicable to EIA) and the schema enforcement happens using the metadata. Testing a new year of data where you are trying to add a new column will often result in that column not showing up your tested output unless you update the schema.

Reply:
I kind of feel like that sentiment is already covered in that bb paragraph? How would you update it to be more clear?